src/NorthwindWebsite/Core/ApplicationSettings/FileUploadOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using NetEscapades.Configuration.Validation;
+using System.ComponentModel.DataAnnotations;
+
+namespace NorthwindWebsite.Core.ApplicationSettings
+{
+    public class FileUploadOptions : IValidatable
+    {
+        [Required]
+        public string ImageFileFormats { get; set; }
+
+        [Required]
+        public int ImageMaxSize { get; set; }
```

Comment:
I am not sure `Required` attribute will work for valuable type: if you have no such parameter in file, value will be automatically set to `0`, if you want to confirm value is not 0, then you should set `Range` attribute or just pickup [my ](https://github.com/dotnet-fizzyy/ASP.Net-Identity/blob/master/src/IdentityWebApi/ApplicationLogic/Validation/DefaultValueAttribute.cs) implementation